### PR TITLE
v2 HackMD CLI

### PR DESCRIPTION
**Pull request by JamesCamel**
- Update commands to be in line with v2 HackMD api
- Bump Oclif to v2
- Add config test CI

### CODE REVIEW

File: test/config.test.ts [hunk 1]
```diff
@@ -38,58 +38,17 @@ describe('Config test', function () {
       .to.throw(/Could not read JSON config file at/)
   })
 
-  it.skip('should throw error if no serverUrl is set', function () {
+  it.skip('should throw error if no hackmdEndpointURL is set', function () {
```
Comment (Yukaii) [inline test/config.test.ts:41]: `hackmdEndpointURL` should be optional. If no value is given, the default endpoint `api.hackmd.io/v1` should be used.

Comment (JamesCamel) [reply test/config.test.ts]: Fixed.

File: src/config.ts [hunk 2]
```diff
@@ -4,38 +4,21 @@ import {homedir} from 'os'
...
-  serverUrl: process.env.HMD_CLI_SERVER_URL || process.env.CMD_CLI_SERVER_URL,
-  enterprise: (process.env.HMD_CLI_COOKIE_PATH || process.env.HMD_CLI_SERVER_URL)
-    ? true
-    : (process.env.CMD_CLI_COOKIE_PATH || process.env.CMD_CLI_SERVER_URL)
-      ? false
-      : toBooleanConfig(process.env.HMD_CLI_ENTERPRISE),
-  loginID: process.env.HMD_CLI_ID || process.env.CMD_CLI_ID,
-  loginPassword: process.env.HMD_CLI_PASSWORD || process.env.CMD_CLI_PASSWORD
+  hackmdAPIEndpointURL: process.env.HMD_API_ENDPOINT_URL,
+  accesstoken: process.env.HMD_API_ACCESS_TOKEN
```
Comment (Yukaii) [inline src/config.ts:21]: `accesstoken` should be written in camelCase:

```
accessToken
```

**Review by Yukaii — CHANGES_REQUESTED** [review]

File: src/config.ts (hunk 2 above)
Comment (JamesCamel) [reply src/config.ts]: Fixed.

**Pull request merged**